llvm/lib/Target/AIE/AIE2InstructionSelector.cpp

```diff
@@ -1628,6 +1633,9 @@ bool AIE2InstructionSelector::selectG_AIE_LOAD_UNPACK(
   Register DstReg = UNPACKI.getOperand(0).getReg();
   Register SignReg = UNPACKI.getOperand(3).getReg();
 
+  if (ShouldAntecipate)
```

Comment:
should it not be `ShouldAnticipate` ? 
Is there a more descriptive name for it? Like `CanJoinLoadAndUse` ?

Reply:
I changed the name and also included a comment.